erts/preloaded/src/erlang.erl

```diff
@@ -946,7 +946,8 @@ float_to_binary(_Float) ->
       Options :: [Option],
       Option  :: {decimals, Decimals :: 0..253} |
                  {scientific, Decimals :: 0..249} |
-                 compact.
+                 compact |
+                 pretty_print.
```

Comment:
The documentation for float_to_binary needs to be updated as well.

Reply:
I saw that the float_to_binary just forwards to float_to_list for a description of the options and you added some examples so I think it is good now.